### PR TITLE
managing students, infinite scroll for also record students

### PR DESCRIPTION
**Pull request by benciks**
Known issues:
- currently because of the infinite scroll, when you update student, it wont get changed in the list even though it does get updated and refetched, gotta find a way
- translations

### CODE REVIEW

File: src/pages/admin/students/[id].vue [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+});
+const { executeMutation: updateStudent } = useMutation(
+  graphql(`
+    mutation updateUser($student: UpdateUserInput!) {
+      updateUser(input: $student) {
+        id
+        firstName
+        lastName
+        role
+        student {
```
Comment (tomhaerter) [inline src/pages/admin/students/[id].vue:54]:
```suggestion
        student {
            id
```

File: src/pages/admin/students/new.vue [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+});
+
+const { executeMutation: createStudent } = useMutation(
+  graphql(`
+    mutation createStudent($student: CreateStudentInput!) {
+      createStudent(input: $student) {
+        id
+        firstName
+        lastName
+        student {
```
Comment (tomhaerter) [inline src/pages/admin/students/new.vue:35]:
```suggestion
        student {
            id
```

**Review by tomhaerter — COMMENTED** [review]
Adding the IDs might help with updating the cache. Either way, we should always fetch the IDs in the frontend

**Comment by tomhaerter** [comment]
Besides, I think it looks good. Apart from the cache not updating, I'd say there's nothing stopping us releasing this.

**Comment by benciks** [comment]
The infinite scroll now has a better implementation, translations were added as well

**Pull request merged**